src/main/java/uk/nhs/hee/tis/revalidation/connection/repository/HideRepository.java

```diff
@@ -0,0 +1,10 @@
+package uk.nhs.hee.tis.revalidation.connection.repository;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+import uk.nhs.hee.tis.revalidation.connection.entity.HideConnectionLog;
+
+@Repository
+public interface HideRepository extends MongoRepository<HideConnectionLog, String> {
```

Comment:
Should be name as HideConnectionRepository.

Reply:
I have changed it. Thanks @uzairbaig